src/components/Trends/GetHypedTrends.js

```diff
@@ -2,18 +2,34 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { compose, pure } from 'recompose'
 import * as actions from './actions.js'
+import { simpleSort } from '../../utils/sortMethods'
 
 class GetHypedTrends extends React.Component {
   render () {
-    const { render, ...rest } = this.props
-    return render(rest)
+    const { render, items, ...rest } = this.props
+    const trends = items.slice(-1)
+    const props = {
+      ...rest,
+      items: trends.length > 0 ? [sortByHype(trends)] : trends
+    }
+    return render(props)
   }
 
   componentDidMount () {
     this.props.fetchHypedTrends()
   }
 }
 
+const sortByHype = items => {
+  const sortedData = items[0].topWords.sort(sort('score'))
+  return {
+    ...items,
+    topWords: sortedData
+  }
+}
+
+const sort = sortBy => (a, b) => simpleSort(a[sortBy], b[sortBy])
```

Comment:
Move this to util file. @haritonasty

Reply:
And find one more place in our code, where we have this function and replace it with util method.